src/08_ecosystem/00_chapter.md

```diff
@@ -0,0 +1,51 @@
+# The Async Ecosystem
```

Comment:
Insert line breaks in this file?

Reply:
Everything should be below the github line limit now and looks fine to me in mdbook serve. Is there anything else you meant by this comment?